src/pytest_workflow/plugin.py

```diff
@@ -286,25 +286,26 @@ class YamlFile(pytest.File):
     """
     This class collects YAML files and turns them into test items.
     """
-
```

Comment:
This class doesn't need to be initialised at all?

Reply:
Nevermind, this is explained in the next comment

Reply:
Init is already defined in the base class.